internal/tfschema/tfschema.go

```diff
@@ -147,7 +147,13 @@ func getResourceSchema(resourceType string, resource hclwrite.Block, dir string,
 	if !ok {
 		providerSchemas = &ProviderSchemas{}
 
-		cmd := exec.Command("terraform", "providers", "schema", "-json")
+		// Use tofu by default (if it exists).
+		name := "terraform"
+		if _, err := exec.LookPath("tofu"); err == nil {
```

Comment:
this was broken for cases where only tofu was installed.
currently always defaults to tofo is exists.

May want to make it configurable (?) - not sure...

Reply:
can you elaborate? broken how? make what configurable?

Reply:
you mean that as long as `tofu` is installed - this would resolve to it, even if someone specifies `defaultToTerraform`? or did you mean something else

Reply:
broken if only tofu was installed and not terraform. (Terratag would not work).
The comment is outdated. If someone uses defaultToTerraform it will default to terraform. I had written this comment before adding 'defaultToTerraform'.

Reply:
and what happens if only terraform is installed, and `defaultToTerraform` isn't provided?

Reply:
it will use tofu.

1. If only terraform is installed - use terraform.
2. if only tofu is installed - use tofu.
3. If both terraform and opentofu are installed - use tofu.
4. If both terraform and opentofu are installed and default terraform is provided - use terraform.

Reply:
so it sounds like there's no bug?

Reply:
before the fix it will always use "terraform" which is a bug if only opentofu is installed.
I don't think there are any bugs after the fix.

Reply:
thanks